src/index.js

```diff
@@ -25,6 +25,7 @@ export default React.createClass({
   propTypes: {
     border: React.PropTypes.bool,
     className: React.PropTypes.string,
+    component: React.PropTypes.oneOf([ 'span', 'i' ]),
```

Comment:
@hassankhan I'm not sure the name "component" makes sense here as in React world that implies an actual component. What about `tagName` or something instead?